src/models/users.js

```diff
@@ -0,0 +1,23 @@
+const userQuery = require('../../queries/users');
+
+const getAllUsers = () => {
+  const user = userQuery.getAllUsers();
+  return user.then(result => result.length < 1 ? { error: 'error retrieving users', status: 500 } : result)
```

Comment:
Does this error mean that there was an issue retrieving users? What about if there are no users?

Reply:
updated error message, resending for approval